README.md

```diff
@@ -1,8 +1,8 @@
 # GraphQL Dotnet Parser
-[![AppVeyor](https://img.shields.io/appveyor/ci/graphql-dotnet-ci/parser.svg)](https://ci.appveyor.com/project/graphql-dotnet-ci/parser)
+
 [![Coverage Status](https://coveralls.io/repos/github/graphql-dotnet/parser/badge.svg?branch=master)](https://coveralls.io/github/graphql-dotnet/parser?branch=master)
```

Comment:
Note - it still points to old builds (2017 year) from `master`, but coverage is triggered now only for `test` job.

Reply:
The `test` job will never run on `master`.  Do you intend to add code coverage generation upon merge also?  Maybe modify the `test` job so that it also runs on `push` to `master` or `develop`.

Reply:
> The test job will never run on master.

I know.

> Do you intend to add code coverage generation upon merge also?

Yes.

> Maybe modify the test job so that it also runs on push to master or develop.

I wanted to get advice on how to do better. I understand that we can make as many files as we like. Many stages in these files will be repeated (checkout/build/test/etc). We can place all actions even in one file to "save steps", but then we get a large and complex file. I would like to make the files simpler, let there be more of them. Then each file will correspond to its own scenario. How do you think?

Reply:
I really like the way it's laid out now.  It seems pretty simple.  As I recommended, I would add a couple lines to this same `test.yml` file to make it run when PRs are merged:

```
  push:
    branches:
      - master
      - develop
```

I would not separate test coverage into a separate workflow -- basically all the steps of producing code coverage reports are necessary for running the tests in the first place.  The only thing you could do is skip running on windows. In that case, sure we can create a separate workflow file.  (Or we can add a bunch of if conditions.)

I might also add some more comments to the workflow file(s) -- but, the workflow steps are already labeled, so even that isn't necessary.  Anyone can read through the file and see exactly what the script is doing.  GitHub Action workflows are very easy to follow.

I certainly would not combine the workflow files we have.  Some of the build steps differ - for example, the nuget package location while setting up .net.  This would make the workflow file overly complex, and it would be harder for someone to understand what's going on.

I don't mind the if conditions for operating system, because the steps all DO run within the workflow -- just some only once, and others twice.  We don't have if conditions based on whether it was a 'pull_request' or 'push' trigger, for instance, which fundamentally changes what steps the workflow runs.

Reply:
done